### ISSUE TITLE
bids metadata extractor - tollerate absent dataset_description.json?

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#2335]
#### What is the problem?
Some datasets, such as coming from fcp-indi bucket, lack dataset_description.json or may be even some other files, but otherwise ok BIDS datasets.  ATM (bleeding edge master 0.9.3-845-gd453ce46) it seems to ignore bids extractor altogether.

Trying on ///hbnssi/RawData with a single anatomical present:
```shell
(git-annex)hopa:~/datalad-meta6-4/hbnssi/RawData[master]git
$> ls -ld dataset*
ls: cannot access 'dataset*': No such file or directory
(dev) 1 10014 ->2.....................................:Fri 23 Mar 2018 13:22:49 EDT:.
(git-annex)hopa:~/datalad-meta6-4/hbnssi/RawData[master]git
$> datalad aggregate-metadata
[INFO   ] Aggregate metadata for dataset /home/yoh/datalad-meta6-4/hbnssi/RawData 
[WARNING] 3971 files have no content present, skipped metadata extraction for them 
[INFO   ] Engage metadata extractors: ['datalad_core', 'nifti1', 'bids'] 
[INFO   ] Update aggregate metadata in dataset at: /home/yoh/datalad-meta6-4/hbnssi/RawData 
aggregate_metadata(ok): /home/yoh/datalad-meta6-4/hbnssi/RawData (dataset)
[INFO   ] Attempting to save 6 files/datasets 
save(notneeded): /home/yoh/datalad-meta6-4/hbnssi/RawData (dataset)                                                                                                                                                               
action summary:
  aggregate_metadata (ok: 1)
  save (notneeded: 1)
(dev) 1 10015.....................................:Fri 23 Mar 2018 13:22:57 EDT:.
(git-annex)hopa:~/datalad-meta6-4/hbnssi/RawData[master]git
$> datalad search --show-keys          
nifti1.cal_max               
nifti1.cal_min
nifti1.datatype
nifti1.dim
nifti1.freq_axis
nifti1.intent
nifti1.magic
nifti1.phase_axis
nifti1.pixdim
nifti1.qform_code
nifti1.sform_code
nifti1.sizeof_hdr
nifti1.slice_axis
nifti1.slice_duration
nifti1.slice_end
nifti1.slice_order
nifti1.slice_start
nifti1.spatial_resolution(mm)
nifti1.toffset
nifti1.vox_offset
nifti1.xyz_unit

```

**Comment by yarikoptic** [comment]
marked as for 0.10.0 milestone so we could cover a bigger portion of the datasets -- this issue is common for those from INDI bucket

**Comment by mih** [comment]
This might already be fixed with #2338

**Comment by yarikoptic** [comment]
since all neuroimaging is stripped in favor of extension, let's just assume that it is fixed ... if I discover otherwise -- I will complain in the corresponding tracker
